go.mod

```diff
@@ -8,29 +8,39 @@ require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-cmd/cmd v1.3.0
 	github.com/go-co-op/gocron v1.9.0
-	github.com/go-kit/kit v0.10.0
+	github.com/go-kit/kit v0.11.0
 	github.com/go-redis/redis/v8 v8.11.0
 	github.com/go-zoo/bone v1.3.0
-	github.com/gofrs/uuid v3.3.0+incompatible
-	github.com/golang/protobuf v1.5.0
-	github.com/golang/snappy v0.0.1
+	github.com/gofrs/uuid v4.0.0+incompatible
```

Comment:
this is a major version upgrade, we should check the release notes to make sure this will not break anything in the other orb services